api/gunicorn.conf.py

```diff
@@ -21,9 +21,6 @@
 # We use 'os.sched_getaffinity(pid)' not 'os.cpu_count()' because it returns only allowable CPUs.
```

Comment:
@jamesbursa I was curious on your opinion of how we use gunicorn compared to past projects.

Past projects we setup gunicorn in our main function, while here we tell gunicorn to run our app via a specific create_app function. This means all of the code that runs in that create_app function is run extra times for each worker. That means code like logging initialization duplicates. This doesn't seem to cause any issues at the moment, but does make the initialization a bit confusing.